src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+
+    @Override
+    public List<Bulldozer> get() {
+        Bulldozer bulldozer1 = new Bulldozer("B1", "red", 10, 10, 10, 10);
+        Bulldozer bulldozer2 = new Bulldozer("B2", "green", 20, 20, 20, 20);
+        Bulldozer bulldozer3 = new Bulldozer("B3", "blue", 30, 30, 30, 30);
```

Comment:
```suggestion
        Bulldozer firstBulldozer = new Bulldozer("B1", "red", 10, 10, 10, 10);
        Bulldozer secondBulldozer = new Bulldozer("B2", "green", 20, 20, 20, 20);
        Bulldozer thirdBulldozer = new Bulldozer("B3", "blue", 30, 30, 30, 30);
```
the same for other producers